Build/bump-version-UnitsNet.Extensions-major.bat

```diff
@@ -0,0 +1,4 @@
+@echo off
+SET scriptdir=%~dp0
```

Comment:
We can certainly do that. However, I do want to confirm: Are you sure? :-) Speaking from my own experience (mostly with internal packages), I find this strategy questionable. You're right in that adding new units will likely be the most typical reason to change the UnitsNet package, but by no means the only one, I daresay.

When you make a change to the core that doesn't effect any of the extensions functionality, you will end up with a release (i.e., a new version) of the extensions package identical (except in version) to the previous one. Likewise, adding a new piece of functionality to the extensions package (say, bringing back another retired piece) will result in a new version of the core package, identical to the previous one (or just won't happen until there is a new release of the core).

The downside of not keeping the versions together is that you may end up in a situation where you use UnitsNet 7.1.5 (for example) and UnitsNet.Extensions 1.5.4 (again - just an example), and they are perfectly compatible, in fact "matching" or "corresponding" versions. This may seem confusing. Again, speaking from experience, it's not as intimidating as it looks :-)

If you think keeping the versions together is the right strategy, we'll be more than happy to implement it. I just wanted to make sure all the relevant facts were considered first :-)

Reply:
We can do either way, I don't have any strong opinions. 

I thought the nuget would primarily be about bringing back the number extension methods since they were left out in v4 due to the much larger binary size it introduced. I'm not sure what other kinds of extension methods will be added to this new nuget, that won't be added to the main library? If the nuget is primarily for number extensions, we could also consider renaming the nuget to `UnitsNet.NumberExtensions` or similar to make that purpose more clear. `Extensions` and the description `Auxiliary functionality for Units.NET` does seem a bit vague to me and I don't currently know what other kind of auxiliary functionality will be added later?

If the library will be about number extensions, then I lean towards following the versioning of the main library since they are so tightly coupled and it's easier to understand what versions are compatible.

If the library will be about a bunch of other extension methods or helper-code that for some reason doesn't fit in the main library, then maybe it warrants having its own versioning, but I'd like to learn what you envision here.

Reply:
The idea was to use this package for any functionality that wouldn't make it into the core library because of size constraints (as discussed in #372; as long as it's feasible to separate it, of course). To be honest, I don't know exactly what ended up being removed - I just know of the number extensions, as my team has been using them. Going forward, you could either bring back other pieces of functionality that have been removed, or introduce brand new pieces. If you find it unlikely that this could ever go beyond just the number extensions, unlikely it would ever become "broader", I'll trust your judgement - let's call it "NumberExtensions" (or similar) and change the description accordingly.

As for versioning: I would still find it preferable to keep them separate, precisely for the reasons cited. If the versions remain "in sync", you will end up with a new version of the extensions (or number extensions) package, even if the change doesn't affect it. (#502 represents an example of such a modification, I think; the removal of Nan/infinity support wouldn't, as far as I can tell, result in any changes to number extensions.) Consequently, you'll have a new version identical to the previous one. Is it a problem? I don't know, but I find it a bit odd to be honest. It's up to you - if you still prefer this (always in sync) approach, we'll incorporate it into our PR.

Reply:
Right, then my preferred solution is getting more clear by now.

1. Rename to nuget/project + test project to `UnitsNet.NumberExtensions`, update description to something like "Adds extension methods to number types to more easily create quantities, such as 5.Meters() instead of Length.FromMeters(5)." \*
2. Keep the versioning scripts
3. Update UnitsNet versioning scripts to also update UnitsNet.Extensions - for now - until we see the need to version them separately. \**
4. Generate test cases as mentioned earlier, to get more accurate test coverage.

\* I would rather have more nugets with specific purposes than one general purpose "everything else" nuget, plus we don't currently know of anything else that would go into it.

\**) We always have the option of going that way later, but not so easily the other way. 95% of the time we release a new UnitsNet nuget it contains a new unit, besides it doesn't really hurt anyone if 1 out of 10-20 versions of UnitsNet.NumberExtensions nuget don't have any real changes. We release very often and most people don't upgrade to every version anyway.

Reply:
We'll incorporate this into our PR sometime next week :-) Thanks!

Reply:
Based on the discussion above, I've submitted another PR.